README.rst

```diff
@@ -735,14 +735,59 @@ The filter can have the following parameters:
   This parameter specifies which character will be used to identify the
   Logstash section.
 
+- ``backslash_escaping_ignore_string='@@@'``
+  This parameter sets a string of characters than can be prepended
+  to a string used in the Logstash configuration specification to prevent
+  backslahes from being escaped in the resulting Logstash pipeline configuration
+  file.
```

Comment:
Please remove `used in the Logstash configuration specification as that's quite redundant in the context.

Also please remove the examples bellow and just add the following at the end of this paragraph:

```
(e.g. `'@@@sshd(?:\[%{POSINT:[system][auth][pid]}\])?:'` will turn to `"sshd(?:\[%{POSINT:[system][auth][pid]}\])?:"` instead of to `"sshd(?:\\[%{POSINT:[system][auth][pid]}\\])?:"`).
```